barretenberg/cpp/src/barretenberg/ultra_honk/ultra_prover.cpp

```diff
@@ -99,9 +94,7 @@ template <UltraFlavor Flavor> void UltraProver_<Flavor>::execute_log_derivative_
 
     if constexpr (IsGoblinFlavor<Flavor>) {
         instance->compute_logderivative_inverse(beta, gamma);
```

Comment:
At the very least, if it does make sense for the instance to be responsible for computing commitments, these methods should have names that indicate that commitments are being computed.